deploy/examples/create-external-cluster-resources.py

```diff
@@ -1467,7 +1484,9 @@ def _gen_output_map(self):
         self.out_map["ROOK_EXTERNAL_FSID"] = self.get_fsid()
         self.out_map["ROOK_EXTERNAL_USERNAME"] = self.run_as_user
         self.out_map["ROOK_EXTERNAL_CEPH_MON_DATA"] = self.get_ceph_external_mon_data()
-        self.out_map["ROOK_EXTERNAL_USER_SECRET"] = self.create_checkerKey()
+        self.out_map["ROOK_EXTERNAL_USER_SECRET"] = self.create_checkerKey(
```

Comment:
remove  the and not self._arg_parser.upgrade from the if else
`if not self._arg_parser.rgw_pool_prefix and not self._arg_parser.upgrade:`